base_station/gui/src/components/Cameras.vue

```diff
@@ -195,37 +205,45 @@
           if (new_index !== 0) {
             this.cams[this.cam_index_1 - 1] = false
```

Comment:
based on previous comments, change from cam_index_1 - 1 to just cam_index_1 and so on